functions/go/set-gcp-resource-ids/pkg/kpt/packagecontext.go

```diff
@@ -6,17 +6,13 @@ import (
 	"github.com/GoogleContainerTools/kpt-functions-sdk/go/fn"
 )
 
-func IsLocalConfig(o *fn.KubeObject) bool {
-	return o.IsLocalConfig()
-}
-
```

Comment:
We have the same `IsLocalConfig` function [in the `go.fn` lib](https://github.com/GoogleContainerTools/kpt-functions-sdk/blob/master/go/fn/object.go#L665).